src/components/veNft/VeNftCarouselItem.tsx

```diff
@@ -34,24 +36,29 @@ const VeNftCarouselItem = ({
     )
   }
 
-  const getCarouselImage = () => {
-    if (!variant) {
-      return undefined
-    }
-    if (isActive) {
-      return tokenMetadata
-        ? tokenMetadata.thumbnailUri
-        : getVeNftBaseImage(baseImagesHash, variant)
-    }
-    return getVeNftBaseImage(baseImagesHash, variant)
-  }
+  const getCarouselImage = useCallback(
+    ({ useFallback }: { useFallback: boolean }) => {
+      if (!variant) {
+        return undefined
+      }
+
+      if (isActive) {
+        return tokenMetadata
+          ? tokenMetadata.thumbnailUri
+          : getVeNftBaseImage(baseImagesHash, variant, { useFallback })
+      }
+      return getVeNftBaseImage(baseImagesHash, variant, { useFallback })
+    },
+    [baseImagesHash, variant, isActive, tokenMetadata],
+  )
 
   return (
     <Col span={8}>
       {variant && (
         <>
           <Image
-            src={getCarouselImage()}
+            src={getCarouselImage({ useFallback: false })}
+            fallback={getCarouselImage({ useFallback: true })}
```

Comment:
Thats dope. Non-blocking, but I guess we could add some better "empty image" state here. Like a grey banana emoji or something.